resources/js/app.js

```diff
@@ -12,16 +32,17 @@
 		})
 	})
 
-	app.controller('TabController', function () {
-		this.tab = 1
-		this.setTab = function (stab) {
-			this.tab = stab
-			$(".button-collapse").sideNav('hide');
-		}
-		this.isSet = function (stab) {
-			return this.tab == stab
-		}
-	})
+    app.controller('TabController', function ($location) {
+        this.tab = "/home";
```

Comment:
@gkrishnan724 super awesome, can you please push the changes, yours is a very important PR 😃